wallet-new/src/Cardano/Wallet/Kernel/DB/AcidState.hs

```diff
@@ -160,15 +157,21 @@ switchToFork n blocks = runUpdateNoErrors $
 --
 --  Given prefiltered utxo's, by account, create an HdAccount for each account,
 --  along with HdAddresses for all utxo outputs.
+--
+-- NOTE: we don't have the slotId of transactions occuring in the Utxo,
+--       which prevents us from recording the BlockMeta for the initial utxo
 createHdWallet :: HdRoot
-               -> Map HdAccountId PrefilteredUtxo
+               -> Map HdAccountId (Utxo,[AddrWithId],[Core.TxId])
```

Comment:
I don't understand this. You added an additional `[Core.TxId]` argument here, but you also add a comment saying we can't use them, and indeed, in the body of this function we don't seem to use it at all, do we?

Reply:
Fixed: The TxIds are useful in other settings, and createHdWallet was carrying that burden here...